proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,63 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime to SDL Core for get real system time during TLS handshake.
+
+
+## Motivation
+
+Now SDL Core does not have access to the real system time so  need an interface to the system in order to request it when it is required 

```

Comment:
@AStasiuk 
I think its better to clarify that "in some cases SDL may not have access to the real system time" as it depends on a platform SDL is integrated to.

Reply:
@dev-gh Fixed in https://github.com/AStasiuk/sdl_evolution/pull/1/files#diff-8c90e61f2f7752dccc9f3a2e657dd2aa